src/components/ImageWithFallback.tsx

```diff
@@ -0,0 +1,12 @@
+import { useState } from 'react'
+import Image, { ImageProps } from 'next/image'
+
+export const ImageWithFallback = ({ fallbackSrc, ...props }: { fallbackSrc: string } & ImageProps) => {
```

Comment:
I am not sure how useful this component is for static websites

Reply:
I'm not sure either, but it seems like a good safety net against any errors.